sklearn/utils/tests/test_param_validation.py

```diff
@@ -60,8 +61,9 @@ class _Estimator(BaseEstimator):
     def __init__(self, a):
         self.a = a
 
+    @_fit_context(prefer_skip_nested_validation=True)
```

Comment:
Shall implement `partial_fit` as well.

Reply:
Why is that ? this object is there for other kinds of tests but it is not used to test the nested validation.